### ISSUE TITLE
OnActivityResult doesn't work after cropping in Xiaomi 4 i

### ISSUE THREAD

**Issue by annng** [Yalantis/uCrop#440]
I tried implement this library after taking photo. I don't meet any issues in other device, but when I try in Xiaomi 4 i, my code after cropping (OnActivityResult) doesn't work and doing nothing

**Comment by cyberformed** [comment]
Dude, check the code and put a break point onActivityResult() method whether it's returning resultCode as 96 or not.

**Comment by p1nkydev** [comment]
@annng Can you show the code where you're calling uCrop? Is your onActivityResult() calling?

**Comment by annng** [comment]
`if (requestCode == UCrop.REQUEST_CROP) {
           if (resultCode == RESULT_OK) {

                Uri file_uri = UCrop.getOutput(data);
              
                try {

                    File newMediaFile = FileUtils.copyFileToDownloads(file_uri);

                    if (newMediaFile.exists()) {

                        String filePath = Uri.fromFile(newMediaFile).toString();

                        strPhoto = filePath;

                    } else {
                        See.toast(getApplicationContext(), "File not found");
                    }
                } catch (Exception e) {
                    e.printStackTrace();
                    See.toast(getApplicationContext(), e.getMessage());
                }
            }
        }`

i got error "No Such File/Directory"

**Comment by AhmadIzaz** [comment]
@cyberformed yes I am facing the same error and I am getting resultCode 96. Did you know how to solve ??

**Comment by p1nkydev** [comment]
@AhmadIzaz Hello, what URI are you getting back from UCrop.getOutput(data)?
